static/sass/_snapcraft_l-application.scss

```diff
@@ -127,64 +107,85 @@
     }
 
     .store-selector__panel {
-      background-color: #2d2d2d;
+      background-color: var(--vf-color-background-hover);
+      border: 1px solid var(--vf-color-border-default);
       left: 0;
-      padding: 0.5rem 0.5rem 0;
       position: absolute;
       top: 2.3rem;
       width: 100%;
       z-index: 1;
     }
 
     .store-selector__list {
-      background-color: #2d2d2d;
       margin: 0;
       padding-left: 0;
     }
 
     .store-selector__item {
-      border-bottom: 1px solid rgba(255, 255, 255, 0.25);
-      padding: calc(0.4rem - 1px) 0;
+      border-bottom: 1px solid var(--vf-color-border-muted);
 
       &:last-child {
         border-bottom: 0;
       }
 
       a {
-        color: $color-x-light;
+        color: var(--vf-color-text-default);
         display: block;
         overflow: hidden;
+        padding: calc(0.4rem - 1px) 0.5rem;
```

Comment:
Such specific calculations could use a line comment with context on what are they calculating and why these specific values. Also, if possible, it should use Vanilla variables instead of hardcoded values (https://vanillaframework.io/docs/settings/spacing-settings#vertical-spacing).

Reply:
Oh, just noticed that this is not a new line, just moved. But overall, the comment still stands (at least in the scope you can provide context on).